src/persistence/Elsa.Persistence.YesSql/Stores/YesSqlStore.cs

```diff
@@ -41,14 +41,14 @@ public virtual async Task SaveAsync(T entity, CancellationToken cancellationToke
         {
             await _semaphore.WaitAsync(cancellationToken);
 
-            try
+            try  
```

Comment:
This file contains changes only in accidentally added whitespace. Let's clean that.